bitnami/moodle/4.2/debian-11/rootfs/opt/bitnami/scripts/libmoodle.sh

```diff
@@ -156,6 +156,23 @@ moodle_initialize() {
         read -r -a extra_args <<<"$MOODLE_INSTALL_EXTRA_ARGS"
         [[ "${#extra_args[@]}" -gt 0 ]] && moodle_install_args+=("${extra_args[@]}")
 
+        # Handle --prefix (table prefix) being overridden via MOODLE_INSTALL_EXTRA_ARGS
+        mdl_prefix="mdl_"
```

Comment:
This can be simplified: 

```suggestion
        mdl_prefix="mdl_"
        for extra_arg in "${extra_args[@]}"; do
            if [[ $extra_arg == --prefix=* ]]; then
                mdl_prefix=${extra_arg#--prefix=}
                break
            fi
        done
```